### ISSUE TITLE
Hamburger menu not working for longer menus on mobile

### ISSUE THREAD

**Issue by adamkucharski** [epiverse-trace/epiversetheme#5]
Please place an "x" in all the boxes that apply
---------------------------------------------

 - [ X] I have the most recent version of this package and R
 - [ X] I have found a bug
 - [ X] I have a [reproducible example](http://reprex.tidyverse.org/articles/reprex-dos-and-donts.html)

--------

On mobile, if a user loads a page with many options in the dropdown hamburger menu (e.g. the [epidemics page](https://epiverse-trace.github.io/epidemics/)), then the items in the lower section of the dropdown cannot be clicked, because when the user scrolls down, the menu disappears. Likely a CSS issue in the library (i.e. pixel constraint) being used to display the current website.

**Comment by Bisaloo** [comment]
I don't think there is a great solution here. Beyond this specific issue, a dropdown with the many options will always be a bit awkward.
A solution may be to get rid of the dropdown and direct users to the articles index page: https://epiverse-trace.github.io/epidemics/articles/ when we get a large number of vignettes.

**Comment by Bisaloo** [comment]
Fixed by https://github.com/r-lib/pkgdown/pull/2716.

**Comment by jayhesselberth** [comment]
@Bisaloo Can you confirm whether this actually fixes the issue for you? It's tedious to test with pkgdown itself because I'd have to create a bunch of dummy vignettes that actually exist.

**Comment by Bisaloo** [comment]
Yes, I confirm it fixes the issue. I tested with Firefox responsive design mode and this is the result while emulating an iPhone SE 2022. In particular, I confirm I can scroll in the articles menu without accidentally scrolling in the main window. From what I've seen, other phone layout give similar results.

New: 

![Screen Shot 2024-07-15 at 09 35 11](https://github.com/user-attachments/assets/ac5c5889-c687-408a-bc85-80b4d58d6748)

**Comment by jayhesselberth** [comment]
Sorry, I should been more clear. Scrolling already worked on mobile browsers. The intent of the PR was to activate scrolling on *non-mobile* browsers in dropdowns with many articles.

**Comment by Bisaloo** [comment]
> Scrolling already worked on mobile browsers. The intent of the PR was to activate scrolling on non-mobile browsers in dropdowns with many articles.

This PR then also indirectly solves the issue describe it, which affected mobile browsers. When the article dropdown was taking the entire screen, you couldn't scroll down in the dropdown without scrolling down on the main page and thereby closing the dropdown.

I confirm this also activated scrolling on non-mobile browsers as you intended:

![image](https://github.com/user-attachments/assets/d75fa5b3-dc64-47d6-a158-ba637cd1a326)
